src/utils/geolocation-helper.ts

```diff
@@ -23,15 +23,14 @@ export namespace GeoLocationHelper {
 
 	/**
 	 * @param options An optional options object that lets you specify the maximum no. of attempts (default: 1),
-	 * the timeout in milliseconds (default: 2000), the maximum age of a possible cached position in milliseconds (default: 2000),
-	 * and whether to disable the error prompt in the LifeSG app (default: false).
+	 * the timeout in milliseconds (default: 2000), the maximum age of a possible cached position in milliseconds (default: 2000)
 	 * A minimum of one attempt will be made, even if maxAttempts is 0.
 	 * @returns A Promise that gets the current location.
 	 * @resolve type: ILocationCoord
 	 * @reject type: GeolocationPositionError | undefined
 	 */
 	export const getCurrentLocation = async (options: IGetCurrentLocationOptions = {}): Promise<ILocationCoord> => {
-		const { maxAttempts = 1, timeout = 2000, maximumAge = 2000 } = options;
+		const { maxAttempts = 3, timeout = 2000, maximumAge = 2000 } = options;
```

Comment:
what's the rationale of changing it? I think the original intention was to offer retry as an option

Reply:
oh I asked Hilman and he mentioned the default should be 3 instead so I updated it.

Retry is also not offered as an option now though. so I should be exposing that instead? what's the previous discussion on this?

Reply:
ok I have checked with Hilman, it appears it's more ideal to have retry enabled by default. (when maxAttempts=1, it's akin to no retry enabled)

however can we update the comments to reflect the new behaviour
```ts
/**
 * @param options An optional options object that lets you specify the maximum no. of attempts (default: 3),
 * the timeout in milliseconds (default: 2000), the maximum age of a possible cached position in milliseconds (default: 2000),
 * A minimum of one attempt will be made, even if maxAttempts is 0.
 * @returns A Promise that gets the current location.
 * @resolve type: ILocationCoord
 * @reject type: GeolocationPositionError | undefined
*/
```